system/security/shared/secloader.hpp

```diff
@@ -35,12 +35,12 @@ class SecLoader
     /// for the given ESP service
     ///
     /// @param  bindingName     Binding name ie 'WsTopology_smc_myesp'
-    /// @param  authCfg         'Authenticate' IPropertyTree from EspService component binding
+    /// @param  bnd_cfg         'Binding' IPropertyTree associated with ESPService
     /// @param  secMgrCfg       'SecurityManager' IPropertyTree from component config file
     ///
     /// @return an ISecManager Security Manager instance
     ///
-    static ISecManager* loadPluggableSecManager(const char * bindingName, IPropertyTree* authCfg, IPropertyTree* secMgrCfg)
+    static ISecManager* loadPluggableSecManager(const char * bindingName, IPropertyTree* bnd_cfg, IPropertyTree* secMgrCfg)
```

Comment:
Should be named bindCfg (mixed case as other args). Also spelling out bind is more descriptive than bnd